### PR TITLE
Add authlib.

### PR DESCRIPTION
**Pull request by markfennema**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/authlib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "authlib" %}
+{% set version = "0.10" %}
+{% set sha256 = "b61c6c6fd230c4ba8602fd85ee9a40e6dc859387699a1cd1f7247c4b109dcc17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/c8/03/6c676d80761f5194ef343679240bb3cfabb4122f9ee1c068ef83fec4ef7f/Authlib-0.10.tar.gz
```
Comment (marcelotrevisani) [inline recipes/authlib/meta.yaml]: Please change this to:
```
url: https://files.pythonhosted.org/packages/c8/03/6c676d80761f5194ef343679240bb3cfabb4122f9ee1c068ef83fec4ef7f/Authlib-{{ version }}.tar.gz
```

Comment (marcelotrevisani) [reply recipes/authlib/meta.yaml]: In that way when the version update our bot will be able to track this recipe and update it

File: recipes/authlib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set name = "authlib" %}
+{% set version = "0.10" %}
+{% set sha256 = "b61c6c6fd230c4ba8602fd85ee9a40e6dc859387699a1cd1f7247c4b109dcc17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/c8/03/6c676d80761f5194ef343679240bb3cfabb4122f9ee1c068ef83fec4ef7f/Authlib-{{ version }}.tar.gz
```
Comment (hajapy) [inline recipes/authlib/meta.yaml]: I would suggest:

`https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.capitalize() }}-{{ version }}.tar.gz`

**Comment by markfennema** [comment]
@hajapy Done.

**Pull request merged**